README.md

```diff
@@ -1,3 +1,4 @@
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-22041afd0340ce965d47ae6ef1cefeee28c7c493a6346c4f15d667ab976d596c.svg)](https://classroom.github.com/a/Hs1RbhHP)
 # docker-deep-dive-2024
 ## Docker deep dive workbook
 
```

Comment:
No answers to followup questions submitted during course time or during on delivery time.